divine/realm.py

```
@@ -0,0 +1,126 @@
+
+        y, x, text = self.__validate_args(args, pully, pullx)
+        self.realm.addstr(y, x, text)
+
+
+    def ask(self, *args, pully=True, pullx=False, request='str'):
+        if len(args) != 0:
+            y, x, text = self.__validate_args(args, pully, pullx)
+            self.write(text, y, x, pully=pully, pullx=pullx)
+
+        match request:
+            case 'str': return self.realm.getstr()
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)